src/entities/activity/lib/types/item.ts

```diff
@@ -460,7 +460,7 @@ export interface RequestHealthRecordDataItem extends ActivityItemBase {
 
 export type RequestHealthRecordDataItemConfig = {
   removeBackButton: boolean;
-  skippableItem: never;
+  skippableItem: boolean;
```

Comment:
For backwards-compatibility with the previous configuration setup (it's not strictly needed, but just to align with my proposed BE fix):

```suggestion
  skippableItem?: boolean;
```

Reply:
Updating 🤓